inst/cuda/dust.cu

```diff
@@ -3,5 +3,5 @@
 #include <dust/device_info.hpp>
 
 cpp11::sexp dust_device_info() {
-  return dust::cuda::device_info<void>();
+  return dust::cuda::device_info<float>();
```

Comment:
Is it correct that this is always float? Might it sometimes be double

Reply:
this is the default value printed in the information, so the size of floats is nicer to see here than the confusing zero. This is purely informational